src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -30,51 +53,69 @@
 
     public static class Metrics {
         public static class Chassis {
-            public static final double TICKS_PER_REVOLUTION = -1; // TODO: CHANGE CONSTS
-            public static final double DIAMETER_INCHES = -1;
-            public static final double CIRCUMFERENCE_INCHES = Metrics.Chassis.DIAMETER_INCHES * Math.PI;
-            public static final double TICKS_PER_INCH = Metrics.Chassis.TICKS_PER_REVOLUTION
-                    / Metrics.Chassis.CIRCUMFERENCE_INCHES;
+            public static final double TICKS_PER_REVOLUTION = -1;
+            public static final double DIAMETER_METERS = -1;
+            public static final double CIRCUMFERENCE_METERS = Metrics.Chassis.DIAMETER_METERS * Math.PI;
+            public static final double TICKS_PER_METER = Metrics.Chassis.TICKS_PER_REVOLUTION
+                    / Metrics.Chassis.CIRCUMFERENCE_METERS;
             public static final double DISTANCE_FRONT_BACK = -1;
             public static final double DISTANCE_SIDE_SIDE = -1;
-            public static final double INCHES_PER_TICK = Metrics.Chassis.CIRCUMFERENCE_INCHES
+            public static final double METERS_PER_TICK = Metrics.Chassis.CIRCUMFERENCE_METERS
                     / Metrics.Chassis.TICKS_PER_REVOLUTION;
         }
     }
 
-    public static class DriveConstants {
-        public static final boolean kGyroReversed = false;
-        public static final double ksVolts = -1;
-        public static final double kvVoltSecondsPerMeter = -1;
-        public static final double kaVoltSecondsSquaredPerMeter = -1;
-        public static final double kTrackwidthMeters = -1;
-        public static final DifferentialDriveKinematics kDriveKinematics = new DifferentialDriveKinematics(
-                kTrackwidthMeters);
-        public static final double kPDriveVel = -1;
-    }
-
-    public static class AutoConstants {
```

Comment:
Why is this removed? Are you sure it won't be used in other parts of the master branch?